### PR TITLE
Rachelzitomer patch 2

### PR DESCRIPTION
**Pull request by rachelzitomer**
Added some additional files and made a small update to bomcal.north data.

### CODE REVIEW

File: inst/raw_data/desktop.ini [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[LocalizedFileNames]
```
Comment (nspope) [inline inst/raw_data/desktop.ini]: Is this file needed?

**Review by nspope — COMMENTED** [review]
Looks good -- could you please add a README.md to the "inst" directory that gives a brief description of what the files are?

File: inst/raw_data/desktop.ini (hunk 1 above)
Comment (rachelzitomer) [reply inst/raw_data/desktop.ini]: No, good catch. I've deleted it.

**Comment by rachelzitomer** [comment]
> Looks good -- could you please add a README.md to the "inst" directory that gives a brief description of what the files are?

I added a Readme.md in the inst directory, can you take a look at your file descriptions for your files (particularly those in scripts) and make any changes needed?
